NewRemarques.md

```diff
@@ -1,14 +1,3 @@
-## Pinned post : 
-- boolean field on post?
-    If we do that it's hard to validate a unique post pinned by user. The only way would be to get all the post and then request the post where pinned is true and take maybe the lastedUpdatedPost in array. 
-# pinnedPosts = post { where : { pinned: true }} orderBy updatedAt Asc 
```

Comment:
That is the good request. But it will be better Desc?

Reply:
I forgot to save my new NewRemarques. This part no longer exist. I've written asc without thinking of. Just to remember to add an orderBy. But yes I think desc would show the latest pinned post.
Should we have a specific route to get the last pinned post u think?